src/video/cocoa/cocoa_v.mm

```diff
@@ -313,21 +300,28 @@ static void QZ_UpdateVideoModes()
 	return { static_cast<uint>(NSWidth(frame)), static_cast<uint>(NSHeight(frame)) };
 }
 
+/**
+ * Are we in fullscreen mode
+ * @return whether fullscreen mode is currently used
+ */
+bool VideoDriver_Cocoa::IsFullscreen()
+{
+	return this->window != nil && ([ this->window styleMask ] & NSWindowStyleMaskFullScreen) != 0;
```

Comment:
I suppose nil is some objective C thing? Should be consistent on nullptr/nil/NULL I suppose

Reply:
Yes, `nil` is Objective-C. It is supposed to be consistent in the sense that Obj-C types are compared with `nil` and C/C++ pointers with `nullptr`.